doc/pyang.1.dbk

```diff
@@ -937,6 +971,108 @@
+          <informalexample>
+            <screen>pyang --generate-sid-file 20000:100 toaster@2009-11-20.yang</screen>
+          </informalexample>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><option>--update-sid-file</option></term>
+        <listitem>
+          <para>
+            Each time new items are added to a YANG module by the introduction of
+            a new revision of this module, its included sub-modules or imported
+            modules, the associated .sid file need to be updated. This is done
+            by using the <option>--update-sid-file</option> option and providing
+            the previous .sid file as argument. This example, this command shows
+            how to generate the file <emphasis>toaster@2009-12-28.sid</emphasis> based
+            on the SIDs already present in <emphasis>toaster@2009-11-20.sid</emphasis>.
+          </para>
+          <informalexample>
+            <screen>pyang --update-sid-file toaster@2009-11-20.sid toaster@2009-12-28.yang</screen>
+          </informalexample>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><option>--check-sid-file</option></term>
+        <listitem>
+          <para>
+            The <option>--check-sid-file</option> option can be used at any time
+            to verify if a .sid file need to be updated. For example:
+          </para>
+          <informalexample>
+            <screen>pyang --check-sid-file toaster@2009-12-28.sid toaster@2009-12-28.yang</screen>
+          </informalexample>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><option>--list_sid</option></term>
+        <listitem>
+          <para>
+            The <option>--list_sid</option> option can be used before any of the
+            previous options to obtains the list of SIDs assigned or validated. For example:
+          </para>
+          <informalexample>
+            <screen>pyang --list-sid --generate-sid-file 20000:100 toaster@2009-11-20.yang</screen>
+          </informalexample>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><option>--extra-sid-range</option></term>
+        <listitem>
+          <para>
+            If needed, an extra SID range can be assigned to an existing YANG module
+            during its update with the <option>--extra-sid-range</option> option. For example,
+            this command generates the file <emphasis>toaster@2009-12-28.sid</emphasis>
+            using the initial range(s) present in <emphasis>toaster@2009-11-20.sid</emphasis>
+            and the extra range specified in the command line.
+          </para>
+          <informalexample>
+            <screen>pyang --update-sid-file toaster@2009-11-20.sid toaster@2009-12-28.yang --extra-sid-range 20100:100</screen>
+          </informalexample>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term><option>count</option></term>
```

Comment:
this is not an option, but an argument to --generate-sid-file.  However, if I understand this correctly, if you give this argument to --generate-sid-file it doesn't actually generate any file, which seems weird.